doc/introduction/contributing.md

```diff
@@ -82,7 +82,7 @@ Ready to contribute code? Here's how to set up Robotoff for local development.
     poetry run pytest tests
     ```
 
-    Before running the test cases make sure you have a database created. Have a look at [settings.py](../../robotoff/settings.py#L131) the default database name, user, and password is
+    Before running the test cases make sure you have a database created. Have a look at [.env](../../robotoff/settings.py#L131) the default database name, user, and password is
```

Comment:
I think this explanation is for non docker user, so the .env is not relevant to them.

Reply:
@alexgarel then what shall I name it?

Also, I have to add the documentation for https://github.com/openfoodfacts/robotoff/pull/693. Shall I add this documentation in this PR? Or make a new one?

Reply:
the documentation for https://github.com/openfoodfacts/robotoff/pull/693 should be in #693 PR !

Reply:
My bad. 
I realized that later. I have added the documentation in #693  PR itself.

Reply:
Hum indeed you put .env but you ling to `settings.py` :-) 

```suggestion
    Before running the test cases make sure you have a database created. Have a look at [robotoff/settings.py](../../robotoff/settings.py#L131) the default database name, user, and password is
```